recipes/sagemaker-tensorflow-serving-container/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sagemaker-tensorflow-serving-container" %}
+{% set version = "1.6.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aws/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 9dfc0044ac16a9c3bcf11cf967fcbf45bce73b99a34bcc9fec2a44fa19647aa6
+
+build:
+  number: 0
+  script:
+    - cp -r $SRC_DIR $SP_DIR/sagemaker_tensorflow_serving_container
```

Comment:
don't you have a ``setup.py`` for this project?

Reply:
@marcelotrevisani unfortunately there is not a setup.py for this project. I couldn't find much documentation on building packages without a setup.py. I based my recipe on this issue https://github.com/conda/conda-build/issues/3166.